protractor_tests/questions.pageObject.js

```diff
@@ -46,11 +46,27 @@ var QuestionsPage = function() {
    */
   var runCodeBtn = element(by.css('.protractor-test-run-code-btn'));
 
+  /**
+   * Performs boilerplate actions to get and set up the questions page.
+   *
+   * @param {object} browser
+   */
+  this.setUp = function(browser) {
```

Comment:
I believe `browser` is global, you shouldn't need to pass it in.

Reply:
so that's what I thought, but if I don't do that it tells me browser isn't defined. Would I have to do this.browser or?

Reply:
That's odd. I just ran it locally after removing the arg, and it passed. (This is client-side.) Are you referring to the server version?

Happy to work through it tomorrow too.

Reply:
huh, weird. I'll try removing it tomorrow and if it works I'll just push.

No, just client version.